examples/with-universal-configuration/package.json

```diff
@@ -3,7 +3,9 @@
   "version": "1.0.0",
   "scripts": {
     "dev": "next",
-    "build": "next build",
+    "clean": "rimraf .\\node_modules\\.cache",
+    "build-dev": "npm run clean && next build --env=development",
```

Comment:
Then can just add another env var.

IMHO that's unrelated to what the example is trying to convey. There's no need for `command-line-args` to get universal config working.